src/test/resources/testcontainers.properties

```diff
@@ -0,0 +1 @@
+testcontainers.reuse.enable=false
```

Comment:
I don't know if is a good idea to start (and stop) one container for test case.
My personal preference would be for leaving the reuse enabled by default, letting some specific tests to override it.